rocketpy/motors/motor.py

```diff
@@ -497,6 +499,21 @@ def propellant_initial_mass(self):
             Propellant initial mass in kg.
         """
 
+    @property
+    def structural_mass_ratio(self):
+        """Calculates the structural mass ratio. The ratio is defined as
+        the dry mass divided by the initial total mass.
+
+        Returns
+        -------
+        float
+            Initial structural mass ratio.
+        """
+        initial_total_mass = self.dry_mass + self.propellant_initial_mass
+        if initial_total_mass == 0:
+            raise ValueError("Motor total mass is zero!")
+        return self.dry_mass / initial_total_mass
```

Comment:
Just to keep in mind: other option is to try/except the result. In case of DivisionByZero error, throw an error. 
This way you would avoid the ir comparison, especially because the comparison against a value (in this case, the ==0) is something really painful in any language.

This is a non-block